proto/metadata.proto

```diff
@@ -279,6 +279,10 @@ message Metadata {
     bool isis_interface_afi_unsupported = 85;
     // Devices don't support modify table entry operation in P4 Runtime.
     bool p4rt_modify_table_entry_unsupported = 86;
+    // Parent of OS component is of type SUPERVISOR or LINECARD
```

Comment:
each of these comments should end with a period

Reply:
thanks, fixed